src/models/Settings.js

```diff
@@ -52,6 +52,7 @@ function (Okta, Errors, BrowserFeatures) {
       'features.rememberMe': ['boolean', true, true],
       'features.rememberDevice': ['boolean', true, true],
       'features.rememberDeviceAlways': ['boolean', true, false],
+      'features.autoPush': ['boolean', true, false],
```

Comment:
if `features.autoPush` is `undefined` or a non-boolean value when `save`, the model.validate will fail. Since there is a `validate` to strict that, I think `!!this.model.get('autoPush')` could be `this.model.get('autoPush') === true`